.github/workflows/examples-linux-standalone.yaml

```diff
@@ -212,7 +212,12 @@ jobs:
                     linux debug lock-app \
                     out/linux-x64-lock/chip-lock-app \
                     /tmp/bloat_reports/
-
+            - name: Build example contact sensor with UI
+              run: |
```

Comment:
This needs a timeout-minutes, right?

Reply:
Should we have one? The only real use of these things I have ever seen was to "well, it times out now, so let's increase it".

I understand tests which seem to timeout sometimes, however builds are very deterministic currently so I would rather not repeat the same "timeout minutes 10" which seems like a random round value anyway.

Reply:
We still have the action timeout, so it will not run forever, just if we have a busy-loop we get feedback in about 1h instead of 10min + whatever time it takes for the preceeding builds. 

Without the timeout though, we are not as likely to timeout just because we get scheduled on slightly slower machines.

Reply:
Right, the main reason for per-step timeouts is precisely to not wait for the whole workflow to time out to discover problems....

Reply:
I don't feel too strongly about this, by the way, hence the approval above.